src/swarmus_ros_navigation/rviz/pioneer_0_config_test.rviz

```diff
@@ -0,0 +1,390 @@
+Panels:
```

Comment:
Can this file have a more relevant name than test? Should it be the default rviz config to use? Is it useful for automated tests?

Reply:
File removed.